src/db/begin_transaction.cr

```diff
@@ -11,12 +11,14 @@ module DB
     # The exception thrown is bubbled unless it is a `DB::Rollback`.
     # From the yielded object `Transaction#commit` or `Transaction#rollback`
     # can be called explicitly.
-    def transaction
+    # Returns the value of the block when no exception was raised.
```

Comment:
Minor: The latter part is technically unnecessary. When the method raises, it generally doesn't have a return value.
```suggestion
    # Returns the value of the block.
```